hpi-schul-cloud/schulcloud-server
# BC-6493 - Disable video upload in tldraw

# Description
Change allowed asset extensions list

QA note:
Change tested on env:
https://bc-6493-disable-tldraw-video-upload.nbc.dbildungscloud.dev/
No issues found.  Criteria ticket have been done.  But in testing, 2 bugs were found which are bugs of tldraw itself.  
1. when selecting a file, it is still possible to select mp4 and webm files - where this should already be deactivated  
2. when adding an mp4 or webm file, the message shows that the extension is wrong, while when adding other illegal files like txt pdf or other we get the message  "laden" and it's nothing happen 
These two bugs will be fixed in the next tickets.



## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-6493
https://github.com/hpi-schul-cloud/tldraw-client/pull/56


## Changes


## Datasecurity


## Deployment


## New Repos, NPM pakages or vendor scripts


## Approval for review


> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.
